jicofo-selector/src/main/kotlin/org/jitsi/jicofo/bridge/colibri/ColibriV2SessionManager.kt

```diff
@@ -325,6 +322,33 @@ class ColibriV2SessionManager(
         }
     }
 
+    private fun expireAndAllocate(session: Colibri2Session, participantInfo: ParticipantInfo): ColibriAllocation {
+        return try {
+            session.expire()
+            session.created = false
+
+            val stanzaCollector = session.sendAllocationRequest(participantInfo)
```

Comment:
I am wary of having a different codepath for this. Can you call `expire` and then return an error so that the allocation is re-tried?

Reply:
I made more local testing and realised that expiration is already part `removeSession(session)` call, so there is no need to add additional codepath. I reverted the change but changed the log level to WARN for that situation as it eventually can restore the conference.